lit_tests/kernel/wave/codegen.py

```diff
@@ -1051,8 +1051,12 @@ def repeat(
         output = torch.zeros(8, 64, 128, dtype=torch.float32)
         print(chained_gemm(q, k, v, output).module_op)
 
-        # CHECK-LABEL:     func.func @chained_gemm(
+        # CHECK-LABEL:     func.func @chained_gemm
+        # CHECK-SAME:        (%[[ARG0:[a-zA-Z0-9_]+]]: !stream.binding
```

Comment:
nit: `[[ARG0:.*]]`

Reply:
I too prefer (and started with) this haha, but for some reason with` :.*` it is failing to find it 🤔

Reply:
Maybe it matched multiple args? You can try to explicitly match all the `chained_gemm` args